src/nupic/regions/TMRegion.cpp

```diff
@@ -793,8 +842,18 @@ void TMRegion::serialize(BundleIO &bundle) {
   f.write((const char*)&args_, sizeof(args_));
   f << columnDimensions_ << " ";
   f << std::endl;
+	// Need to save the output buffers
+  f << "outputs [";
```

Comment:
why does this need to be added now?

Reply:
I was flip-flopping about where to serialize the output buffers.  Most Region implementations do it new so I added it to TMRegion to be consistent.

However for Cereal I think I will move this to the Region serialization where it can be done all in one place and cover all region implementations.